packages/core/src/controllers/TransactionsController.ts

```diff
@@ -52,7 +52,8 @@ export const TransactionsController = {
         account: accountAddress,
         projectId,
         cursor: state.next,
-        onramp
+        onramp,
+        cache: onramp === 'coinbase' ? 'no-cache' : 'default'
```

Comment:
Maybe a comment explaining why would be helpful here? Also `undefined` might be a little better

Reply:
Addresses 🙏